### PR TITLE
disambiguate segment and scope meta-variable

### PR DESCRIPTION
**Pull request by o-**

### CODE REVIEW

File: doc/ir-semantics.text [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 x        variables
 
-P ::= Segment  active   ↦ S, (Ds ↦ S)*   program:            active version and others
-S ::= start ↦ i,          I              segment:            instruction stream with dedicated start
+P ::= Segment  active   ↦ M, (Ds ↦ M)*   program:            active version and others
```
Comment (mhyee) [inline doc/ir-semantics.text:5]: Do you still want to keep `Ds` and little `s`? (And where does `M` come from?)

Comment (o-) [reply doc/ir-semantics.text:5]: M is defined bellow.

Comment (mhyee) [reply doc/ir-semantics.text:5]: Sorry, I meant why `M` was chosen. The definition below is fine.

Comment (o-) [reply doc/ir-semantics.text:5]: ah segMent :/ do you have a better idea?

Comment (mhyee) [reply doc/ir-semantics.text:5]: I don't have a better idea, but I think it's reasonable. I was just curious.

**Comment by o-** [comment]
thanks for the comments.

I am really sorry for making a mess. I'll push this commit to #70 instead, then we can fix all of it in one go.

File: doc/ir-semantics.text (hunk 1 above)
Comment (o-) [reply doc/ir-semantics.text:5]: btw. I use `T` for segmenT now, since it avoids confusion with Module and also I found Dm to look silly.

**Pull request closed without merging**